# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../libdbi-drivers/libdbi-drivers-0.8.3-1-mysql55-header.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- libdbi-drivers-0.8.3-1/drivers/mysql/dbd_mysql.c.orig	2011-04-13 10:02:21.820077817 +0200
+++ libdbi-drivers-0.8.3-1/drivers/mysql/dbd_mysql.c	2011-04-13 10:02:29.172077718 +0200
@@ -51,7 +51,7 @@
 #include <dbi/dbi-dev.h>
 #include <dbi/dbd.h>
 
-#include <mysql/mysql.h>
+#include <mysql.h>
 #include "dbd_mysql.h"
 
 static const dbi_info_t driver_info = {
